doc/api/esm.md

```diff
@@ -643,12 +643,12 @@ The resolve hook returns the resolved file URL and module format for a
 given module specifier and parent file URL:
 
 ```js
-import { URL, pathToFileURL } from 'url';
```

Comment:
`URL` is actually used in the function below. I might miss something but I think this should stay as it is? The same applies to the other entry.

Reply:
It's already in scope. No need to import it.

"The `URL` class is also available on the `global` object." https://nodejs.org/api/url.html#url_class_url

Reply:
It turns out that the type checkers would prefer if it were imported, so I will revert this change.

Reply:
I actually retract the above statement as I was able to find a way that did not necessitate importing globals.